site/docs/voice/rateLimits.mdx

```diff
@@ -1,6 +1,6 @@
 ---
```

Comment:
does the url need to change?

Reply:
Seemed safer to me to leave the URL and just update the title.